docs/content/configuration/upstream.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Configuring Upstream Proxy to use Go modules repository
+description: How to Fetch modules directly from a Go modules repository such as GoCenter
+weight: 1
+---
+
+The upstream proxy used when fetching modules directly is by default the actual source, e.g github.com. This can be configured to use a Go modules repository like GoCenter.
+
+1. Create a filter file (e.g /usr/local/lib/FilterForGoCenter) with letter `D` (stands for "direct acccess") in first line. For more details, please refer to documentation on  - [Filtering Modules](/configuration/filter)
```

Comment:
could you put the /usr/local/lib/FilterForGoCenter inside backticks, so that it gets formatted as "code" ?

i.e. `/usr/local/lib/FilterForGoCenter`